types/resemblejs/index.d.ts

```diff
@@ -1,96 +1,151 @@
-// Type definitions for Resemble.js v1.3.0
+// Type definitions for Resemble.js v3.2.5
 // Project: https://github.com/rsmbl/Resemble.js
 // Definitions by: Tim Perry <https://github.com/pimterry>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+declare global {
+    type Buffer = Uint8Array;
```

Comment:
Have you tested this with the node types?

I just tried it locally and it doesn't seem to work for me, I get: `Cannot redeclare block-scoped variable 'Buffer'`. I think you have to use `interface`. Two interfaces declarations will merge, whilst two `type` declarations for the same name will fail.

Also this should probably have a comment explaining why we're doing this rather than just importing node.

Reply:
No, I haven't tested it, sorry. I fixed everything as you suggested. Take a look, please.